tests/databrowser.js

```diff
@@ -481,5 +481,119 @@ define([
+                                },
+                            },
+                        },
+                    },
+                }, {
+                    id: "casedb",
+                    uri: "jr://instance/casedb",
+                    path: "/casedb/case",
+                    name: 'Cases',
+                    structure: {
+                        name: {},
+                    },
+                    subsets: [{
+                        id: "case",
+                        name: "child",
+                        key: "@case_type",
+                        structure: {
+                            dob: {},
+                            invalid: {}
+                        },
+                    }, {
+                        id: "commcare-user",
+                        name: "user",
+                        key: "@case_type",
+                        structure: {
+                            code_name: {},
+                            user_role: {},
+                        }
+                    }],
+                }];
+
+            before(function (done) {
+                util.init({
+                    plugins: plugins,
+                    javaRosa: {langs: ['en']},
+                    core: {
+                        dataSourcesEndpoint: function (callback) { callback(CASE_AND_USER_DATA); },
+                        invalidCaseProperties: ['invalid'],
+                        onReady: function () {
+                            databrowser.initDataBrowser(this);
+                            dataTree = this.$f.find(".fd-external-sources-tree").jstree(true);
+                            done();
+                        }
+                    }
+                });
+            });
+
+            it("should drag/drop case property", function (done) {
+                util.loadXML("");
+                var mug = util.addQuestion("DataBindOnly", "mug"),
+                    input = $("[name=property-calculateAttr]"),
+                    editor = input.ckeditor().editor,
+                    widget = util.getWidget('property-calculateAttr');
+                widget.input.promise.then(function () {
+                    editor.on('change', function() {
+                        assert.equal(mug.p.calculateAttr, "#case/dob");
+                        done();
+                    });
+                    assert(!mug.p.calculateAttr, "unexpected: " + mug.p.calculateAttr);
+                    util.findNode(dataTree, "dob").data.handleDrop(input);
+                });
+            });
+
+            it("should drag/drop user property", function (done) {
+                var form = util.loadXML(""),
+                    mug = util.addQuestion("DataBindOnly", "mug"),
+                    input = $("[name=property-calculateAttr]"),
+                    editor = input.ckeditor().editor,
+                    widget = util.getWidget('property-calculateAttr');
+                widget.input.promise.then(function () {
```

Comment:
I see some tests do that and [some](https://github.com/dimagi/Vellum/blob/b8714350b636b5a4a05fed9d7a5d8dd5c912f815/tests/databrowser.js#L119) do it this way. In this case I wanted to reset the editor since I'm dragging a different value into the same property widget in the previous test. Is there something I'm overlooking?